auth/src/main/java/com/firebase/ui/auth/viewmodel/email/RecoverPasswordHandler.java

```diff
@@ -29,4 +30,18 @@ public void onComplete(@NonNull Task<Void> task) {
                     }
                 });
     }
+
+    public void startCustomReset(@NonNull final String email, @NonNull ActionCodeSettings actionCodeSettings) {
```

Comment:
This method is almost exactly the same as `startReset` could it be combined?  We could make the `actoinCodeSettings` argument `@Nullable` and then do something like this:

```java
Task<Void> reset = actionCodeSettings != null 
  ? getAuth().sendPasswordResetEmail(email, actionCodeSettings)
  : getAuth().sendPasswordResetEmail(email)

reset.addOnCompleteListener(...
```